python/protoc_gen_validate/validator.py

```diff
@@ -22,6 +21,15 @@
     "HEADER_STRING": r'^[^\u0000\u000A\u000D]*$'
 }
 
+
+class UnimplementedException(Exception):
```

Comment:
I was able to go further. The only time this was raised was for `no_sparse` assertions which weren't relevant in python (`no_sparse validation is not implemented because protobuf maps cannot be sparse in Python`) and look like aren't relevant anywhere: https://github.com/dkunitsk/protoc-gen-validate/blob/main/tests/harness/executor/cases.go#L1156

So I removed this exception and no_sparse will now pass validation silently.